Pipfile

```diff
@@ -20,7 +20,7 @@ flake8-black = "==0.1.0"
 flake8-bugbear = "==19.8.0"
 flake8-docstrings = "==1.3.1"
 flake8-quotes = "==2.1.0"
-sphinx = "==2.1.2"
+sphinx = "==2.3.1"
```

Comment:
Why this change?

Reply:
It seems that there was an update to a dependency of sphinx (probably unpinned) that broke the doc build process. It's fixed in this update